.devcontainer/Dockerfile

```diff
@@ -42,9 +42,9 @@ RUN echo INSTALL DOTNET RUNTIME 2.1 \
 && apt-get install -y dotnet-runtime-2.1
 
 # # install bot tools
-RUN npm i -g @microsoft/botframework-cli
+RUN npm i -g --unsafe-perm @microsoft/botframework-cli
```

Comment:
There's some issues when `@microsoft/botframework-cli` tries to install `node-pre-gyp`. After searching the [`node-pre-gyp` repo](https://github.com/mapbox/node-pre-gyp#readme), common belief seems to be that there's a permissions issue with npm folders. I tried several suggested solutions and the only one that worked was [the one used here](https://github.com/mapbox/node-pre-gyp/issues/433#issuecomment-471028255). This may be fixed if `@microsoft/botframework-cli` updates their `node-gyp`/`node-pre-gyp` dependencies.